tools/install.sh

```diff
@@ -369,25 +376,44 @@ if [ -z "$KING_PHISHER_SKIP_SERVER" ]; then
 	if [ "$KING_PHISHER_USE_POSTGRESQL" == "yes" ]; then
 		if [ -f "/etc/init.d/postgresql" ]; then
 			/etc/init.d/postgresql start &> /dev/null
+			service postgresql status | grep online | grep -v grep &> /dev/null
+			if [ ! $? -eq 0 ]; then
+				postgresql-setup --initdb &> /dev/null
+				/etc/init.d/postgresql start &> /dev/null
+			fi
+		fi
+		if [ -f "/usr/lib/systemd/system/postgresql.service" ]; then
+			systemctl start postgresql &> /dev/null
+			systemctl is-active postgresql | grep active | grep -v grep &> /dev/null
```

Comment:
`systemctl is-active` will return a non-zero number if the server is inactive. Use that instead of grepping. Additionally should probably check if the service is active before attempting to start it.